localisation/wc_religions.csv

```diff
@@ -267,8 +267,8 @@ STAT_PIETY_PIETY_CHI;Chi;Chi;Chi;;Chi;;;;;;;;;x
 MONTHLY_PIETY_PIETY_CHI;Monthly Chi Increase;Gain mensuel de Chi;Monatlicher Chi-Anstieg;;Aumento mensual de Chi;;;;;;;;;x
 pandaren_religion_group_FEATURES;�Y*�! Has a hard time converting other [pandaren_religion_group.GetGroupName] religions;;�Y*�! Hat es schwer, andere [pandaren_religion_group.GetGroupName]-Religionen zu konvertieren;;;;;;;;;;;x
 MONK;monk;;M�nch;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+suntouched;Suntouched;;;;;;;;;;;;;x
+suntouched_DESC;The Suntouched are a specific branch of Titan worship that originated in Uldum as a primary sect of Amathet tol'vir. They pride themselves on protecting the works of the Titans and rely on solar magic to smite their enemies. Due to their fanatical nature, they mostly see entities that are not affiliated with the Titans as corrupted by Void's influence and therefore must be purged from the cosmos.;;;;;;;;;;;;;x
```

Comment:
Altered the last sentence, to correct a minor mistake and clear up some confusion. As I understand it, all Amathet believe everyone else is tainted by the void and needs to be destroyed, not just most.
```suggestion
suntouched_DESC;The Suntouched are a specific branch of Titan worship that originated in Uldum as a primary sect of Amathet tol'vir. They pride themselves on protecting the works of the Titans and rely on solar magic to smite their enemies. Due to their fanatical nature, they see entities that are not affiliated with the Titans as tainted by the Void's influence, and therefore unworthy of existence.;;;;;;;;;;;;;x
```